### PR TITLE
Set baseline expectations for categorical cross-entropy

### PR DESCRIPTION
**Pull request by qualiaMachine**
In the same spirit as the regression episode, I thought it might be useful to establish a baseline expectation in terms of the categorical cross-entropy loss metric. You can calculate the expected performance of a model that guesses at random by using log(n), where n is the number of classes

### CODE REVIEW

**Review by svenvanderburg — CHANGES_REQUESTED** [review]
I like the idea @qualiaMachine. IMO, it makes sense to introduce baselines in episode 3. I see episode 2 as quickly going through the deep learning cycle once, without going into details. Then episode 3 introducing greatly expanding the cycle with more advanced concepts.

If you do want to introduce baselines in episode 2 I would suggest we add a callout box and compute the baseline accuracy instead of categorical crossentropy loss, to make things a little bit more intuitive for people without a mathemical background.

What do you think?

**Comment by qualiaMachine** [comment]
Thank you for your comment, @svenvanderburg! I agree that a baseline measure in terms of accuracy is probably more intuitive. However, I would argue that it's useful for real-world deep learning practitioners to know the equation necessary for establishing a baseline using cross-entropy loss. It's a very common loss metric that comes up in all kinds of classification problems. Even if they don't understand the math fully, it can be useful thing to memorize down the line and can help you detect problems while the model is still training. In contrast, interpreting the confusion matrix / accuracy is intuitive enough that I'm not sure it's worth a callout.

**Comment by svenvanderburg** [comment]
@qualiaMachine Agree that it is useful to know. Although I worked on deep learning for many years with just an intuitive understanding of crossentropy without the mathematics 🙈😂. 

Maybe it's an idea that we introduce it in episode 4? There was also use categorical crossentropy if I am not mistaken. That we we keep episode 2 relatively clean and not overwhelm students.

Otherwise I suggest to put the addition that you currently have in a callout box, and add a little bit more context. I think to fully understand why the loss would be `log(n)` you need some more explanation of the mathematics.

**Comment by qualiaMachine** [comment]
I totally get the desire to keep episode 2 light. The only thing that has me wanting to stick to episode 2 is that's where we introduce categorical cross-entropy. I think my explanation of the baseline math might make more sense if I also write a little paragraph unpacking categorical cross-entropy loss in episode 2. I can probably do that all as a callout box if that seems most appropriate? I could also concede and stick to episode 4 if you'd really like -- I won't die on this hill haha.

**Comment by svenvanderburg** [comment]
OK, let's go for a callout box explaining crossentropy loss and introducing the baseline loss in episode 2. We can always move it to 4 if it doesn't work.

**Comment by carschno** [comment]
Just to follow-up (cf. #476): is this still relevant?
